var/spack/repos/builtin/packages/openfast/package.py

```diff
@@ -86,7 +90,20 @@ def cmake_args(self):
                     '-DHDF5_USE_STATIC_LIBRARIES=ON',
                 ])
 
+        if '+openmp' in spec:
+            options.extend([
+                '-DOPENMP:BOOL=ON',
+            ])
+            
         if 'darwin' in spec.architecture:
             options.append('-DCMAKE_MACOSX_RPATH:BOOL=ON')
 
         return options
+    
+    def flag_handler(self, name, flags):
+        spec = self.spec
+        if name in ['cflags', 'cxxflags', 'cppflags', 'fflags']:
+            if '+openmp' in spec:
+                flags.append('-fopenmp')
```

Comment:
Use `self.compiler.openmp_flag` instead of `-fopenmp`.

Reply:
Great! Didn't know about that one. Made the change. Thanks!